vmdb/spec/models/ems_refresh/refreshers/ec2_refresher_spec.rb

```diff
@@ -85,7 +85,9 @@ def assert_ems
     @ems.vms_and_templates.size.should    == 46
     @ems.vms.size.should                  == 27
     @ems.miq_templates.size.should        == 19
-    @ems.orchestration_stacks.size.should == 2
+    @ems.orchestration_stacks.size.should == 1
+
+    @ems.orchestration_stacks_with_nested.size.should == 2
```

Comment:
In this test setting we have one root level stack and it has one child stack. `orchestration_stacks` used to return all stacks, the number was 2. Now it returns only the root stack. New method `orchestration_stacks_with_nested` instead returns all stacks.